smartsim/_core/control/controller.py

```diff
@@ -135,6 +135,10 @@ def start(
             # it may be called seperately
             self.poll(5, True, kill_on_interrupt=kill_on_interrupt)
 
+    @property
+    def active_orch_dict(self) -> t.Dict[str, Job]:
```

Comment:
I'm not sure that I'd put the return type in the function name.  Maybe "active_orchestrator_jobs" would be better?

Reply:
Changed name